model/license.go

```diff
@@ -77,10 +77,12 @@ type TrialLicenseRequest struct {
 }
 
 type Features struct {
-	Users                     *int  `json:"users"`
-	LDAP                      *bool `json:"ldap"`
-	LDAPGroups                *bool `json:"ldap_groups"`
-	MFA                       *bool `json:"mfa"`
+	Users      *int  `json:"users"`
+	LDAP       *bool `json:"ldap"`
+	LDAPGroups *bool `json:"ldap_groups"`
+	MFA        *bool `json:"mfa"`
+
+	// MM-47384: "OpenId", "GoogleOAuth", "Office365OAuth" keys are obsolete. These will soon be removed from license
 	GoogleOAuth               *bool `json:"google_oauth"`
 	Office365OAuth            *bool `json:"office365_oauth"`
 	OpenId                    *bool `json:"openid"`
```

Comment:
This seems to be the pattern used [in Golang source code](https://cs.opensource.google/go/go/+/master:src/compress/flate/inflate.go;l=46?q=deprecated&ss=go%2Fgo):
```suggestion
	// Deprecated: This feature will be removed from the license because it's available without a license.
	GoogleOAuth               *bool `json:"google_oauth"`
	
	// Deprecated: This feature will be removed from the license because it's available without a license.
	Office365OAuth            *bool `json:"office365_oauth"`

	// Deprecated: This feature will be removed from the license because it's available without a license.
	OpenId                    *bool `json:"openid"`
```

It's a bit repetitive but necessary for [Godoc](https://pkg.go.dev/compress/flate#ReadError) and IDEs.

Reply:
Thanks!